### PR TITLE
LPS-189317 Only prevent drop event for images (files), not for portlets.

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Issue: https://liferay.atlassian.net/browse/LPS-189317

When the Page Comments portlet is in a widget page, the rest of the porlets does not work via drag&drop. This is because when there is an editor, we are preventing the drag&drop events. The origin of this is https://issues-archive.liferay.com/browse/LPS-87348 

My proposal is to prevent this drop event inly when you are dropping images (more generic, files), so it does not affect to portlets. 

Let me know your thoughts on this. Thanks!

### CODE REVIEW

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Review by markocikos — COMMENTED** [review]
@ambrinchaudhary The approach to solution looks good to me! A minor tweak / question inline.

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/ckeditor.jsp [hunk 1]
```diff
@@ -183,7 +183,9 @@ name = HtmlUtil.escapeJS(name);
 		var validDropTarget =
 			element.isContentEditable || !!element.getAttribute('droppable');
 
-		if (!validDropTarget) {
+		var droppedFiles = event._event.dataTransfer.files || 0;
```
[the remark this answers is not in the document]
Comment (ambrinchaudhary) [reply modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/ckeditor.jsp]: You are totally right, I've just changed it. Thanks!

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Review by markocikos — APPROVED** [review]
@ambrinchaudhary LGTM! :sparkles:

**Comment by markocikos** [comment]
ci:forward

**Pull request closed without merging**